apps/rush-lib/src/utilities/JsonSchemaValidator.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import Validator = require('z-schema');
+import * as os from 'os';
+import * as path from 'path';
+import JsonFile from './JsonFile';
+
+export type ValidateErrorCallback = (errorDescription: string) => void;
+
+export default class JsonSchemaValidator {
```

Comment:
Good suggestion -- I forgot about that. :-)

---
In reply to: [116288366](https://github.com/Microsoft/web-build-tools/pull/181#discussion_r116288366) [](ancestors = 116288366)